regression/contracts/assigns_enforce_conditional_unions/test.desc

```diff
@@ -0,0 +1,24 @@
+CORE
+main.c
+--enforce-contract update _ --pointer-check --pointer-overflow-check --signed-overflow-check --unsigned-overflow-check --conversion-check
+^\[is_high_level.\d+\] line 52 Check that latch is assignable: SUCCESS$
+^\[is_high_level.\d+\] line 53 Check that once is assignable: SUCCESS$
+^\[update.\d+\] line 85 Check that POINTER_OBJECT\(\(void \*\)state->digest.high_level.first.ctx->digest\) is assignable: SUCCESS$
+^\[update.\d+\] line 86 Check that state->digest.high_level.first.ctx->flags is assignable: SUCCESS$
+^\[update.\d+\] line 89 Check that POINTER_OBJECT\(\(void \*\)state->digest.high_level.second.ctx->digest\) is assignable: FAILURE$
+^\[update.\d+\] line 90 Check that state->digest.high_level.second.ctx->flags is assignable: FAILURE$
+^\[update.\d+\] line 96 Check that POINTER_OBJECT\(\(void \*\)state->digest.high_level.first.ctx->digest\) is assignable: FAILURE$
+^\[update.\d+\] line 97 Check that state->digest.high_level.first.ctx->flags is assignable: FAILURE$
+^\[update.\d+\] line 100 Check that POINTER_OBJECT\(\(void \*\)state->digest.high_level.second.ctx->digest\) is assignable: SUCCESS$
```

Comment:
This one is failing in regression tests right now.

Reply:
The tool is right and the test case is badly specified, fixing it in a new update.